dietpi/dietpi-software

```diff
@@ -4166,6 +4173,57 @@ WantedBy=multi-user.target
 _EOF_
 		fi
 
+		if To_Install 141 adsb-feeder # ADSB Feeder
```

Comment:
```suggestion
		if To_Install 141 adsb-setup adsb-docker # ADSB Feeder
```
That's the services which shall be enabled. All enabled services are started at the end automatically, but on a fresh install, they are not automatically enabled (unless its is part of the APT package `postinst` script).